### DISCUSSION TITLE
O que você verifica para saber se um site é de ótima qualidade ou não?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#427] [Discussão]
Quando alguém te pergunta

## Verifica se este site tem qualidade?

Qual é a primeira coisa que você verifica?
E depois você verifica o que mais?

**Comment by batoreh** [comment]
Depende de vários quesitos, pois a pergunta por natureza é muito vaga.

Mas admitindo que é com relação a questões de usabilidade, há alguns padrões que podem ser tomados como parâmetro, geralmente chamados de heurísticas.
O mais famoso padrão é o de Nielsen. O mais prático e usual é o do Dr. Pete. Seguem links para maior entendimento.

- [10 Heurísticas de Nielsen](http://blog.caelum.com.br/10-heuristicas-de-nielsen-uma-formula-pra-evitar-erros-basicos-de-usabilidade/)
- [25-point by Dr. Pete](http://drpete.co/blog/25-point-website-usability-checklist)

**Comment by brunoravanhani** [comment]
Para dar minha opinião pessoal seria:

1. **O design**, antes mesmo de clicar em algo ou escrever, tem-se a primeira impressão visual, nesse ponto da para considerar as cores, disposição dos elementos/componentes, fonte, imagens, etc. Isso fora o carregamento inicial da parte visivel da página.
2. **A usabilidade**, depois e olhar, a gente começa a clicar basicamente, então da pra avaliar se tem bugs, se é usável, tem boa performance, bons formulários, bons "Action Buttons", enfim, se eu consigo usar facilmente o site.
3. **A arquitetura**, pra finalizar, tacar a mão to código, verificar se tem boa arquitetura, se é fácil para dar manutenção, se é semântico, se tem boa SEO, se favorece a acessibilidade, se o CSS e o JS não é igual a um pega varetas.

É claro que isso é visão de uma pessoa, que no caso é técnica, então acredito que seria necessário também avaliar se a UX favorece o usuário que é o alvo principal do site, então haveria a necessidade de pesquisas com os usuários, se não houver.

**Comment by felquis** [comment]
@brunoravanhani @netoguimaraes avaliação técnica e usando o DevTools

**Comment by felipefialho** [comment]
Meu olhar cai direto no Design e na Usabilidade. E aqui também inclui, e muito, velocidade de carregamento.

DevTools, linguagens, arquitetura de código.... Tudo isso é coisa muito nossa, de desenvolvedor.

Então, quando inverto e vou acessar algum site / aplicativo, tento acessar como usuário comum.

Só costumo abrir o DevTools e inspecionar, se ver alguma funcionalidade diferente, que me atiça a curiosidade de como foi feita.
